guides/static/guides/css/guide_list.css

```diff
@@ -14,3 +14,15 @@
 .no-underline {
     text-decoration: none;
 }
+
+#search-button{
+    position: relative;
+    right: -535px;
```

Comment:
This is a bit weird - on Safari it seems to work fine, on Firefox, it shifts off the screen when the width gets too small, meaning this probably breaks on mobile devices. Try making your browser window very narrow and you'll see what I mean:
![image](https://user-images.githubusercontent.com/22679924/41378694-2a6d67ba-6f60-11e8-896e-30ec8d246adf.png)

![image](https://user-images.githubusercontent.com/22679924/41378697-2dbb580a-6f60-11e8-8f68-044f37fa8735.png)

Instead of specifying the position like this, shouldn't the button be positioned within the body container?